content/getting-started/events-api-best-practices/best-practices.md

```diff
@@ -0,0 +1,88 @@
+# Best practices to use the Events API with your connector
+
+<!--
+- Explication rapide de l'events api + lien vers la référence doc de l'events api + helpcenter
+- Explication rapide d'un use-case d'une extension connecté à l'events api
+  - aka, mon site ecommerce aimerait se sync avec le pim rapidement aprés des modifications sur un produit
+  -->
+
+This page presents best practices and recommendations you might need while implementing the Events API inside an API based connector.
+
+First, if you need help to setup a connector to use the Events API take a loot at the [Quick start my first webhook](/getting-started/quick-start-my-first-webhook/welcome.html) guide.
+
+## Configure your server
+
+TODO
+
+<!--
+- recommandations devops (voir avec Pierre-yves)
+  - DNS, Firewall, TLS certificate (+lib TLS)
+  - TLS car la data doit être sécurisé entre le PIM et l'extension
+  - https://api.akeneo.com/events-documentation/limits-and-scalability.html#extension-side-limits
+-->
+
+## Handle API events asynchronously
+
+### Connector scalability
+
+The Events API can send a lot of requests to your connector endpoint. The default value being 4000 requests per hour and each one may contain up to 10 events (see [limits and scalability](/events-documentation/limits-and-scalability.html#limit-of-event-api-requests-per-hour)).
```

Comment:
```suggestion
The Events API can send a lot of requests to your connector endpoint. It can reach up to 4000 requests per hour, and each one may contain up to 10 events (see [limits and scalability](/events-documentation/limits-and-scalability.html#limit-of-event-api-requests-per-hour)).
```